### PR TITLE
imagestream name change

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: templates/osbs-buildroot-imagestream.yml.j2 [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: ImageStream
 metadata:
   annotations:
     openshift.io/image.insecureRepository: {{ osbs_insecure_repository }}
-  name: osbs-buildroot
+  name: osbs-rcm-buildroot
```
Comment (lcarva) [inline templates/osbs-buildroot-imagestream.yml.j2]: Com to think of it, this should probably be a variable.

Comment (rcerven) [reply templates/osbs-buildroot-imagestream.yml.j2]: ok

**Comment by twaugh** [comment]
Do we need to document this variable name somewhere or give it a default?

**Comment by rcerven** [comment]
gave it default, and check in the task

**Pull request merged**